tests/utils/GraphicalUnitTester.py

```diff
@@ -76,7 +76,7 @@ def _load_data(self):
             The pre-rendered frame.
         """
         frame_data_path = os.path.join(
-            os.path.join(self.path_control_data, "{}.npz".format(str(self.scene)))
+            os.path.join(self.path_control_data, f"{str(self.scene)}.npz")
```

Comment:
The call to `str` within the f-string can be removed:

```suggestion
            os.path.join(self.path_control_data, f"{self.scene}.npz")
```

Reply:
Good catch!